### PR TITLE
Create ShreyashSomvanshi.json

### PR DESCRIPTION
**Pull request by ShreyashSomvanshi**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4938"></a>

### CODE REVIEW

**Review by iamthecloverly — APPROVED** [review]

File: data/ShreyashSomvanshi.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "name": "Shreyash Somvanshi",
+  "type": "personal",
+  "bio": "Artificial Intelligence & Data Science Engineer | Open Source Beginner | Technical Content Writer ",
+  "tags": ["Data Science", "AI", "Open Source", "Content Writer"],
+  "socials": [
+    { "icon": "FaGithub", "url": "https://github.com/ShreyashSomvanshi" },
+    { "icon": "SiMedium", "url": "https://medium.com/@ShreyashSomvanshi" }
+  ],
+  "testimonials": ["None...."],
```
Comment (kumarsonsoff3) [inline data/ShreyashSomvanshi.json:10]: It would be better to remove this section if you currently don't have a testimonial. 
```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/ShreyashSomvanshi.json (hunk 1 above)
Comment (ShreyashSomvanshi) [reply data/ShreyashSomvanshi.json]: Updated! @kumarsonsoff3

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Review by iamthecloverly — APPROVED** [review]

**Review by iamthecloverly — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Welcome to LinkFree 🎉

**Comment by ShreyashSomvanshi** [comment]
Thank you All reviewers and maintainers!

**Pull request merged**